src/main/java/io/specto/hoverfly/junit/dsl/RequestedServiceBuilder.java

```diff
@@ -0,0 +1,21 @@
+package io.specto.hoverfly.junit.dsl;
+
+import io.specto.hoverfly.junit.dsl.matchers.PlainTextFieldMatcher;
+
+public class RequestedServiceBuilder extends AbstractServiceBuilder {
```

Comment:
I think it is OK to be different for stubbing and verification. If you are using Mockito, you are more likely to do exact mocking, and then fuzzy verification. When you are more likely to mock a exact method call to return a specific response, but only verify the call is invoked with any arguments. The same applies to HTTP stubbing. Stubbing should be exact, and verification can be fuzzy.